cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ReplicaDistributionAbstractGoal.java

```diff
@@ -181,18 +186,18 @@ protected void updateGoalState(ClusterModel clusterModel, Set<String> excludedTo
     // Log broker Ids over balancing limit.
     // While proposals exclude the excludedTopics, the balance still considers utilization of the excludedTopic replicas.
     if (!_brokerIdsAboveBalanceUpperLimit.isEmpty()) {
-      LOG.warn("Replicas count on broker ids:{} {} above the balance limit of {} after {}.",
-          _brokerIdsAboveBalanceUpperLimit, (_brokerIdsAboveBalanceUpperLimit.size() > 1) ? "are" : "is",
-          _balanceUpperLimit,
-          (clusterModel.selfHealingEligibleReplicas().isEmpty()) ? "rebalance" : "self-healing");
+      LOG.info("Replicas count on broker ids:{} {} above the balance limit of {} after {}.",
```

Comment:
Nit: Can we make this one `debug` rather than `info` -- i.e. because it specifies a soft goal violation in a usual rebalance, and we already show if the goal is `FIXED` or `VIOLATED` after the rebalance. This level of detail might be more appropriate for a debug-level.